pinot-common/src/main/proto/worker.proto

```diff
@@ -40,6 +40,7 @@ message CancelResponse {
 message QueryRequest {
   repeated StagePlan stagePlan = 1;
   bytes metadata = 2; // Serialized Properties
+  string planVersion = 3;
```

Comment:
Suggest making it int and rename it to `version`. This version can also be applied to metadata in case we change its format